docs/running-on-yarn.md

```diff
@@ -461,15 +461,24 @@ To use a custom metrics.properties for the application master and executors, upd
   </td>
 </tr>
 <tr>
-  <td><code>spark.yarn.security.tokens.${service}.enabled</code></td>
+  <td><code>spark.yarn.security.credentials.${service}.enabled</code></td>
   <td><code>true</code></td>
   <td>
-  Controls whether to retrieve delegation tokens for non-HDFS services when security is enabled.
-  By default, delegation tokens for all supported services are retrieved when those services are
+  Controls whether to obtain credentials for services when security is enabled.
+  By default, credentials for all supported services are retrieved when those services are
   configured, but it's possible to disable that behavior if it somehow conflicts with the
-  application being run.
+  application being run. For the details please see
+  [Running in a Secure Cluster](running-on-yarn.html#running-in-a-secure-cluster)
   <p/>
-  Currently supported services are: <code>hive</code>, <code>hbase</code>
+  Currently supported services are: <code>hdfs</code>, <code>hive</code>, <code>hbase</code>
```

Comment:
This sentence is now a little misleading; should probably either not mention this, or say that these are the built-in services.